dump.go

```diff
@@ -64,6 +71,9 @@ func (s *dumpState) dumpType(v reflect.Value) {
 	} else if s.homePackageRegexp != nil {
 		typeName = s.homePackageRegexp.ReplaceAllLiteralString(typeName, "")
 	}
+	if s.config.Compact {
+		typeName = compactTypeRegexp.ReplaceAllString(typeName, "$1")
+	}
```

Comment:
I wonder if we should keep pointer comments in, but change them to something like `/* p0 */`. When litter is used for "hashing" two different structures with the same pointer structure, but aliasing different values could look identical without them

Reply:
Good idea, fixed.